### PR TITLE
Module assembly

### PR DESCRIPTION
**Pull request by wraftus**
Calibration data from module assembly

### CODE REVIEW

File: odrive/configs/MX_M2.json [hunk 1]
```diff
@@ -0,0 +1,499 @@
+{
+    "axis0": {
```
Comment (wmmc88) [inline odrive/configs/MX_M2.json]: what MX?

**Comment by wmmc88** [comment]
@wraftus see #12 for can id allocations

File: odrive/configs/MX_M2.json (hunk 1 above)
Comment (wraftus) [reply odrive/configs/MX_M2.json]: we don't know yet, depends on what ends up going on that odrive

File: odrive/configs/template.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
             "enable_step_dir": false,
             "enable_watchdog": false,
             "startup_closed_loop_control": false,
```
Comment (wmmc88) [inline odrive/configs/template.json:13]: should be true eventually

File: odrive/configs/template.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
-            "startup_encoder_index_search": false,
+            "startup_encoder_index_search": true,
             "startup_encoder_offset_calibration": false,
             "startup_homing": false,
```
Comment (wmmc88) [inline odrive/configs/template.json:16]: should be true eventually

File: odrive/configs/template.json [hunk 4]
```diff
@@ -221,7 +221,7 @@
             "enable_step_dir": false,
             "enable_watchdog": false,
             "startup_closed_loop_control": false,
```
Comment (wmmc88) [inline odrive/configs/template.json:223]: should be true eventually

File: odrive/configs/template.json [hunk 5]
```diff
@@ -221,7 +221,7 @@
...
-            "startup_encoder_index_search": false,
+            "startup_encoder_index_search": true,
             "startup_encoder_offset_calibration": false,
             "startup_homing": false,
```
Comment (wmmc88) [inline odrive/configs/template.json:226]: should be true eventually

**Comment by wmmc88** [comment]
I updated ids in [qpup](https://github.com/q-pup/qpup_robot/commit/1960c9bd5402d36fddeb2c71bdfd114d14a96b0b)

File: odrive/odrive_sheet.csv [hunk 6]
```diff
@@ -1,7 +1,7 @@
 Odrive Number,Location,Axis0 CAN ID,Axis0 Motor ID,Axis0 Motor Location,Axis1 CAN ID,Axis1 Motor ID,Axis1 Motor Location
-0,Front-Left,1,,,2,,
-1,Front-Centre,3,,,4,,
-2,Front-Right,5,,,6,,
+0,Front-Left,1,M10,FLK,2,M12,FLH
```
Comment (wmmc88) [inline odrive/odrive_sheet.csv:2]: NIT match the location names to the odrive json names

**Review by matthew-reynolds — APPROVED** [review]
stonks

**Review by wmmc88 — APPROVED** [review]

**Pull request merged**